* EvoSuite 1.0.6
* Going to generate test cases for class: org.jfree.chart.plot.XYPlot
* Starting client
* Connecting to master process on port 14953
* Analyzing classpath: 
  - /tmp/run_evosuite.pl_78508_1545149528/build
  - /tmp/run_evosuite.pl_78508_1545149528/lib/servlet.jar
* Finished analyzing classpath
* Generating tests for class org.jfree.chart.plot.XYPlot
* Test criterion:
  - Branch Coverage
* Setting up search algorithm for whole suite generation
* Total number of test goals: 912
* Using seed 1000
* Starting evolution
[MASTER] 08:12:22.458 [logback-2] ERROR TestCluster - Failed to check cache for java.util.LinkedList<E> : Type points to itself
[MASTER] 08:12:22.489 [logback-2] ERROR TestCluster - Failed to check cache for java.util.HashMap<K, V> : Type points to itself
[MASTER] 08:13:01.725 [logback-2] ERROR ClientNodeImpl - Cannot inform master of change of state
java.rmi.ServerException: RemoteException occurred in server thread; nested exception is: 
	java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
	java.io.OptionalDataException
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:389) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport$1.run(Transport.java:200) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport$1.run(Transport.java:197) ~[na:1.8.0_191]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport.serviceCall(Transport.java:196) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:573) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:834) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:688) ~[na:1.8.0_191]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:687) ~[na:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_191]
	at sun.rmi.transport.StreamRemoteCall.exceptionReceivedFromServer(StreamRemoteCall.java:283) ~[na:1.8.0_191]
	at sun.rmi.transport.StreamRemoteCall.executeCall(StreamRemoteCall.java:260) ~[na:1.8.0_191]
	at sun.rmi.server.UnicastRef.invoke(UnicastRef.java:161) ~[na:1.8.0_191]
	at java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:227) ~[na:1.8.0_191]
	at java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:179) ~[na:1.8.0_191]
	at com.sun.proxy.$Proxy7.evosuite_collectStatistics(Unknown Source) ~[na:na]
	at org.evosuite.rmi.service.ClientNodeImpl.updateStatistics(ClientNodeImpl.java:237) ~[evosuite-current.jar:1.0.6]
	at org.evosuite.statistics.StatisticsSender.sendIndividualToMaster(StatisticsSender.java:68) [evosuite-current.jar:1.0.6]
	at org.evosuite.statistics.StatisticsListener$1.run(StatisticsListener.java:71) [evosuite-current.jar:1.0.6]
Caused by: java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
	java.io.OptionalDataException
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:348) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport$1.run(Transport.java:200) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport$1.run(Transport.java:197) ~[na:1.8.0_191]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_191]
	at sun.rmi.transport.Transport.serviceCall(Transport.java:196) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:573) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:834) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:688) ~[na:1.8.0_191]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_191]
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:687) ~[na:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_191]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_191]
Caused by: java.io.OptionalDataException: null
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1593) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:431) ~[na:1.8.0_191]
	at java.util.HashMap.readObject(HashMap.java:1407) ~[na:1.8.0_191]
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_191]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_191]
	at java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1170) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2178) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2069) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1573) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2287) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2211) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2069) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1573) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:431) ~[na:1.8.0_191]
	at java.util.HashMap.readObject(HashMap.java:1407) ~[na:1.8.0_191]
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_191]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_191]
	at java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1170) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2178) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2069) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1573) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2287) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2211) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2069) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1573) ~[na:1.8.0_191]
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:431) ~[na:1.8.0_191]
	at sun.rmi.server.UnicastRef.unmarshalValue(UnicastRef.java:322) ~[na:1.8.0_191]
	at sun.rmi.server.UnicastServerRef.unmarshalParametersUnchecked(UnicastServerRef.java:628) ~[na:1.8.0_191]
	at sun.rmi.server.UnicastServerRef.unmarshalParameters(UnicastServerRef.java:616) ~[na:1.8.0_191]
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:338) ~[na:1.8.0_191]
	... 12 common frames omitted
* Search finished after 102s and 291 generations, 176795 statements, best individual has fitness: 361.8333333333333
* Minimizing test suite
[MASTER] 08:14:06.935 [logback-2] ERROR TestCluster - Failed to check cache for java.util.Enumeration<E> : Type points to itself
* Going to analyze the coverage criteria
* Coverage analysis for criterion BRANCH
* Coverage of criterion BRANCH: 57%
* Total number of goals: 912
* Number of covered goals: 516
* Generated 195 tests with total length 854
* Resulting test suite's coverage: 56%
* Generating assertions
* Resulting test suite's mutation score: 16%
* Permissions denied during test execution: 
  - java.io.FilePermission: 
         write /tmp/+~JT4753765966339704530.tmp: 1
  - java.lang.RuntimePermission: 
         writeFileDescriptor: 1
* Writing JUnit test case 'XYPlot_ESTest' to evosuite-branch
* Done!

* Computation finished
* EvoSuite 1.0.6
* Going to generate test cases for class: org.jfree.chart.plot.CategoryPlot
* Starting client
* Connecting to master process on port 14953
* Analyzing classpath: 
  - /tmp/run_evosuite.pl_78508_1545149528/build
  - /tmp/run_evosuite.pl_78508_1545149528/lib/servlet.jar
* Finished analyzing classpath
* Generating tests for class org.jfree.chart.plot.CategoryPlot
* Test criterion:
  - Branch Coverage
* Setting up search algorithm for whole suite generation
* Total number of test goals: 717
* Using seed 1000
* Starting evolution
[MASTER] 08:15:07.520 [logback-2] ERROR TestCluster - Failed to check cache for java.util.LinkedList<E> : Type points to itself
* Search finished after 101s and 362 generations, 233750 statements, best individual has fitness: 137.5
* Minimizing test suite
* Going to analyze the coverage criteria
* Coverage analysis for criterion BRANCH
* Coverage of criterion BRANCH: 75%
* Total number of goals: 717
* Number of covered goals: 540
* Generated 151 tests with total length 637
* Resulting test suite's coverage: 75%
* Generating assertions
* Resulting test suite's mutation score: 17%
* Permissions denied during test execution: 
  - java.io.FilePermission: 
         write /tmp/+~JT8934377519636360805.tmp: 1
  - java.net.URLPermission: 
         *:* http://www.someFakeButWellFormedURL.org/fooExample: 1
* Writing JUnit test case 'CategoryPlot_ESTest' to evosuite-branch
* Done!

* Computation finished
